packages/react-native-attestation/src/index.ts

```diff
@@ -93,3 +93,12 @@ export const isPlayIntegrityAvailable = async (): Promise<boolean> => {
 
   return Attestation.isPlayIntegrityAvailable();
 };
+
+export const getAppStoreReceipt = async (): Promise<string | undefined> => {
+  if (Platform.OS !== 'ios') {
+    throw new Error('getAppStoreReceipt is only available on iOS');
+  }
+
+  const result = await Attestation.getAppStoreReceipt();
+  return result ?? undefined;
```

Comment:
question: when does getAppStoreReceipt return nullish?

I guess what i'm asking is if this should be throwing an error or if we should return a null value instead.

Reply:
when it's a non-app store local build. updated to just return null. I don't think it should throw if it's just null, we can let the project using the package decide if they want to throw in that case